docs/features/promotions.md

```diff
@@ -5,6 +5,23 @@ Currently you can apply a percentage or flat amount promotion for each product i
 1. Enable the _Orchard Core Commerce - Promotion_ feature.
 2. Edit the product in the content item editor.
 3. Set either the _Discount Percentage_ or _Discount Amount_ fields. You can't set both.
```

Comment:
Just FYI: This might be not related to this PR but when using custom taxes and a discount, the information can be misleading. Also going back to the products page doubles the `Gross Price*`. Here the price is 4$ I have a 10 % discount (not global) and a custom tax with 10%:
![problemm](https://user-images.githubusercontent.com/80963259/217141408-83dbbf4d-81df-40a6-b35d-c5cbbb434324.gif)